content/child/blink_platform_impl.cc

```diff
@@ -741,6 +741,7 @@ struct DataResource {
 const DataResource kDataResources[] = {
     {"missingImage", IDR_BROKENIMAGE, ui::SCALE_FACTOR_100P},
     {"missingImage@2x", IDR_BROKENIMAGE, ui::SCALE_FACTOR_200P},
+    /*
```

Comment:
comment out is not right way if we consider other platforms like Linux. I'd suggest we use define and use USE_MINIMUM_RESOURCES macro.